versions.gradle.kts

```diff
@@ -1,4 +1,4 @@
-extra["slf4j_version"] = "1.7.32"
-extra["log4j_version"] = "2.18.0"
+extra["slf4j_version"] = "2.0.0"
+extra["log4j_version"] = "2.18.1-SNAPSHOT"
```

Comment:
Can I suggest you move back to using Log4J `2.18.0` (instead of the snapshot) and change the `log4j-slf4j-impl` dependency to `log4j-slf4j18-impl` then it _should_ work 😄 
See here for info on the new implementation: https://logging.apache.org/log4j/2.x/log4j-slf4j-impl/index.html
> log4j-slf4j18-impl should be used with SLF4J 1.8.x releases or newer.

Note: `log4j-slf4j18-impl` has a dependency on SLF4J 1.8-beta4 - I don't know why log4j haven't updated the dependency to `2.0.0` yet.

Reply:
Correction, log4j are in the process of upgrading to slf4j 2.x and will introduce `log4j-slf4j2-impl` in 2.19.0 which will depend on SLF4J 2.0.0 and will fully implement the interface
So this PR will have to wait until that happens, sorry for the confusion...

Reply:
Apologies for the confusion. I tried 2.18.1-SNAPSHOT to see if the snapshot would work(against the CI) but I'll put it back to a non-snapshot version as soon as we hear back from log4j

Reply:
Thanks, I think the Log4J SLF4J2 support is handled by this ticket in log4j: https://issues.apache.org/jira/browse/LOG4J2-3370 which is targeted at 2.19.0 - still no ETA on a 2.19.0 release though